packages/select/index.js

```diff
@@ -55,6 +55,11 @@ export class WarpSelect extends kebabCaseAttributes(WarpElement) {
     this._options = this.innerHTML;
   }
 
+  firstUpdated() {
+    // autofocus doesn't seem to behave properly in Safari and FireFox, therefore we set the focus here:
+    this.autoFocus && !this.disabled && !this.readOnly && this.shadowRoot.querySelector('select').focus();
```

Comment:
Does the component need to check if the other properties are falsy or should it be user's responsibility?

Reply:
Hmm good question! 🤔 Yeah maybe it's enough if we just check for autoFocus here

Reply:
Removed `!this.disabled && !this.readOnly` in this commit: [de8c1c1](https://github.com/warp-ds/elements/pull/176/commits/de8c1c1db40fd7119aa48699fe6401ad51589de0)